### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by nayshajain**

### CODE REVIEW

File: package-lock.json
Comment (ridham-virufy) [inline package-lock.json:1]: is there a reason package-lock.json is changed?

Comment (a-wong-8) [reply package-lock.json:1]: lol

Comment (ridham-virufy) [reply package-lock.json:1]: Haha..!! We all were here at one point of life

**Comment by gevuong** [comment]
Changes have been made to address issues. Please re-review PR.

**Review by gevuong — APPROVED** [review]

File: package-lock.json
Comment (nayshajain) [reply package-lock.json:1]: @ridham-virufy  @a-wong-8 Thanks for pointing that out! The change in package-lock.json might have happened due to different versions of npm or an update in dependencies. I was trying to see if I can revert it back but I think @gevuong resolved it. Thanks a lot guys!

**Pull request merged**